games-sports/speed-dreams/speed_dreams-2.2.3.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="Open Motorsport Simulator"
+DESCRIPTION="Speed Dreams is a 3d cross-platform, open source motorsport simulation and racing game.
+
+Speed Dreams is a fork of the open racing car simulator Torcs, aiming to implement exciting new \
+features, cars, tracks and AI opponents to make a more enjoyable game for the player, as well as \
+constantly improving visual and physics realism."
```

Comment:
Keep the lines to a max of 80 characters (I know it's a left over from previous versions) :)
See: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description

Reply:
I fixed up the description, but I was not sure about breaking up SOURCE_URI, CHECKSUM_SHA256 to less than 80 lines?

Reply:
Only the DESCRIPTION is involved here, the others are OK

Reply:
Fixed Description